server/routes/viewingLicences/handlers/viewCases.ts

```diff
@@ -63,7 +63,7 @@ export default class ViewAndPrintCaseRoutes {
         }
       }),
       CaViewCasesTab,
-      showAttentionNeededTab,
+      showAttentionNeededTab: cases.some(c => c.tabType === 'ATTENTION_NEEDED'),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Probably lack of TS/JS understanding but will some return only some of them and do we not need all of them?

Reply:
Some in JS will return true if any of the element matches the condition.
It's a boolean used to show and hide attention needed tab.